src/sass/components/_external-links.scss

```diff
@@ -0,0 +1,27 @@
+a.external-link,
+a.external-link-white,
+.case-studies .hero-component a,
```

Comment:
Could some of these more general selectors be making some links external links when they aren't? This could be better handled on the back-end side of things so the links come in with the classes already and we don't have to worry about it on the front-end.

Reply:
@wullaski The reason for the .case-studies .hero-component a -> is making all links show the icon, this is because the links placed in the content on strapi would always be an external link. Any links that are internal are placed in other components on the page. So the css only targets the links coming from the strapi content.